topic-operator/src/main/java/io/strimzi/operator/topic/model/TopicOperatorException.java

```diff
@@ -19,7 +19,9 @@ public enum Reason {
         /** Kafka error wrapper. */
         KAFKA_ERROR("KafkaError"),
         /** Internal server error. */
-        INTERNAL_ERROR("InternalError");
+        INTERNAL_ERROR("InternalError"),
+        /** Invalid configuration. */
+        INVALID_CONFIG("InvalidConfig");
```

Comment:
This enum is used for classifying the cases of `TopicOperatorException`. Because `INVALID_CONFIG` is never actually used to throw an `TopicOperatorException` I don't think it makes sense to add it to the enum. I think we should just add a constant string in BTC for the reason we used when creating the warning.

Reply:
I did the change, but forgot to remove this line that is not used anymore. Sorry.